Update jobs README (#10)
binaris/jobs: README.md

  company-related is _open_.  Everyone can touch any part of the
  production system, and we believe in adding safeguards to help us
  avoid mistakes when we do so.

> - **Empowerment:** We believe that the most important quality in developers, is taking ownership of their work. Whether implementing a new feature or fixing a bug, we each own the entire life cycle of the task. This includes everything from design, to coding, testing, and integration, all the way up to deployment and ops.

Reviewer: ```suggestion
- **Empowerment:** We believe in our abilities.  Each of us handles the entire life
  cycle of a task, whether implementing a new feature or fixing a bug: from design,
  to coding, testing, and integration, all the way up to deployment and ops.
```
(Also @avnerbraverman) WDYT?  I think this way I shorten it, add confidence in each other, and avoid scare words like "ownership" and "most important".

  features whenever possible, not only in official "releases".  We
  also iterate over our specification and design processes, especially
  after we think we've made a mistake.
- **Innovation:** We've already built a blazing fast Functions-as-a-Service platform with the unique characteristics needed to run ShiftJS well.
  There's nothing like it in the world.  Now we're doing the same for
  integrated development of frontend and backend code.

> - **Empowerment:** We believe that the most important quality in developers, is taking ownership of their work. Whether implementing a new feature or fixing a bug, we each own the entire life cycle of the task. This includes everything from design, to coding, testing, and integration, all the way up to deployment and ops.

Reviewer 2: I think this should be the first bullet or second bullet (right after trust).

Author: moved after trust
